src/stacker-responsibilities.md

```diff
@@ -1,8 +1,21 @@
 # Stacker responsibilities
-The stackers are gaining a lot of responsibilities in sBTC. This chapter should outline the new role of the stackers, and also provide a high level interaction to further details on how this is implemented.
+One of the most significant changes to accomodate the sBTC design is that Stackers must now perform active work to continue receiving PoX rewards. Stackers collectively maintain a BTC wallet for the duration of each reward cycle in which their STX are locked. This chapter outlines the new role of the stackers, and how they interact with each other to fulfill their duties.
 
-Important areas to introduce:
-- The sBTC wallet and how stackers maintain it (introducing FROST)
-- How stackers communicate (Signing protocol + StackerDB)
+- sBTC Wallet address generation
+- sBTC Wallet handoff
+- sBTC Withdrawal fulfillment
```

Comment:
I think we can drop this list now as it does not follow naturally from the previous paragraph.